openedx/features/course_experience/__init__.py

```diff
@@ -22,6 +22,9 @@
 # Waffle flag to enable the sock on the footer of the home and courseware pages
 DISPLAY_COURSE_SOCK = CourseWaffleFlag(WAFFLE_FLAG_NAMESPACE, 'display_course_sock')
 
+# Waffle flag to enable a review page link from the unified home page
+SHOW_REVIEWS_TOOL = CourseWaffleFlag(WAFFLE_FLAG_NAMESPACE, 'show_reviews_tool')
```

Comment:
I'd like all the flags to be named FOO_FLAG so that it is clearer in usage. I see that DISPLAY_COURSE_SOCK also didn't follow this convention.